packages/workflow/src/Workflow.ts

```diff
@@ -378,26 +394,27 @@ export class Workflow {
 			return returnArray;
 		}
 
-		const returnData: any = {}; // tslint:disable-line:no-any
+		const returnData: any = {}; // eslint-disable-line @typescript-eslint/no-explicit-any
 
-		for (const parameterName of Object.keys(parameterValue || {})) {
-			returnData[parameterName] = this.renameNodeInExpressions(parameterValue![parameterName], currentName, newName);
+		for (const parameterName of Object.keys(parameterValue ?? {})) {
+			returnData[parameterName] = this.renameNodeInExpressions(
+				parameterValue![parameterName],
+				currentName,
+				newName,
+			);
 		}
 
 		return returnData;
 	}
 
-
-
 	/**
 	 * Rename a node in the workflow
 	 *
 	 * @param {string} currentName The current name of the node
 	 * @param {string} newName The new name
 	 * @memberof Workflow
 	 */
-	renameNode(currentName: string, newName: string) {
-
+	renameNode(currentName: string, newName: string): void {
```

Comment:
`void` is inferrable. Shouldn't this be caught by the no-inferrable-types rule instead of added in?

Reply:
this rule: `'@typescript-eslint/explicit-module-boundary-types': 'error'` is enforcing that every function signature has to follow the TS standard, meaning if it does not return anything it has to state `void`

I think the inferrable -types rule just applies for arguments and variables